src/opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/NumericalAquifers.cpp

```diff
@@ -145,4 +145,32 @@ namespace Opm {
         }
         return cells;
     }
+
+    std::array<std::set<size_t>, 3> NumericalAquifers::transToRemove(const EclipseGrid& grid) const {
+        std::array<std::set<size_t>, 3> trans;
+        for (const auto& [id, aquifer] : this->m_aquifers) {
+            auto trans_aquifer = aquifer.transToRemove(grid);
```

Comment:
This will give an unused variable warning for the `id` variable:
```
(void)id;
```
should silence it; personally I like to use `_` as symbol for such captured-but-not-to-be-used variables.

Reply:
I did not manage to get the warning with `-Wall -Wextra`, but it can be a matter of the compiler support. 

Do we use `[[maybe_unused]]` in opm in practice now?

Reply:
> Do we use [[maybe_unused]] in opm in practice now?

It is not in my tool belt; but sure - feel free.